src/main/scala/io/phdata/retirementage/loadgen/LoadGenerator.scala

```diff
@@ -27,13 +43,71 @@ object LoadGenerator {
     * @param payloadBytes Payload size
     * @return The test dataframe
     */
-  def generateTable(numRecords: Int, payloadBytes: Int): DataFrame = ???
+  def generateTable(spark: SparkSession, numRecords: Int, payloadBytes: Int): DataFrame = {
+
+    /**
+      * Create the following schema:
+      *   -- id: String (nullable = false)
+      *   -- payload: String (nullable = false)
+      *   -- date: String (nullabe = false)
+      */
+    val schema = StructType(StructField("id", StringType, false) :: Nil)
+      .add(StructField("payload", StringType, false))
+      .add(StructField("date", StringType, false))
+    var dataBuffer = ListBuffer[List[String]]()
+    // Create a string with specified bytes
+    var i          = 0
+    var byteString = ""
+    for (i <- 1 to payloadBytes) {
+      byteString = byteString + "a"
+    }
+    // Create numRecords rows of data with a random UUID and attached a payload of specified bytes and a date
+    var j = 0
+    for (j <- 1 to numRecords) {
+      dataBuffer += List(UUID.randomUUID().toString.substring(0, 9),
```

Comment:
Creating the dataframe from a list will cause issues at large record sized (billions of records). It might be better to 
- create a list of n records
- convert it to a dataframe
- union the dataframe to itself until the desired size is reached